build/docker/installation-package/installation-package.dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM kubeedge/build-tools:latest as builder
+FROM kubeedge/build-tools:1.17.13 as builder
```

Comment:
If we have some other fix in build-tools.dockerfile, the tag  `1.17.13` should be modified? :)

Reply:
yes, if we want update golang version, all related should be fixed. This is the same as the golang version

Reply:
`1.17.13-ke1   1.17.13-ke2  1.18.1-ke1` maybe is a solution.

Reply:
Can the basic image of kubeedge/build-tools be changed to alpine? There are too many vulnerabilities in ubuntu, and the official version has not been repaired @fisherxu